libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -180,6 +180,25 @@ void IRGeneratorForStatements::endVisit(VariableDeclarationStatement const& _var
 				declare(m_context.addLocalVariable(*decl));
 }
 
+bool IRGeneratorForStatements::visit(Conditional const& _conditional)
+{
+	_conditional.condition().accept(*this);
+
+	string condition = expressionAsType(_conditional.condition(), *TypeProvider::boolean());
+	auto type = _conditional.annotation().type;
+	IRVariable toAssign(m_context.newYulVariable(), *type);
+	define(toAssign) << m_utils.zeroValueFunction(*type) << "()\n";
```

Comment:
```suggestion
	declare(toAssign);
```
Or is there really a need to initialize this to zero? It will be assigned to in both cases below anyways, won't it?